src/App.js

```diff
@@ -1,25 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import { Switch, Route } from "react-router-dom";
+import MovieDetails from "./Components/MovieDetailsPage/MovieDetails";
+import MainPage from "./Components/MainPage/MainPage";
+import Header from "./Components/MainPage/Header/Header";
 
-function App() {
+const App = () => {
+  const APIKEY = "3aece1730ec57334758cdeb57c0d6adb";
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <p>heeeellllloooo...</p>
-          <h1>hi</h1>
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Header title="MOVIE DATABASE" />
```

Comment:
If having something as uppercase is a design choice, I would recommend transforming the text via CSS instead:

```css
text-transform: uppercase;
```

Then you can ensure that it always be an uppercase no matter what is the input.